### ISSUE TITLE
Bug in conversion

### ISSUE THREAD

**Issue by KenanSulayman** [js2coffee/js2coffee#367]
`0..constructor.constructor('', 'console.log(0..constructor.constructor("", "return 1;")())')()`

will produce the same in the right pane; will cause an error when clicked since it's not compliant nor valid CoffeeScript.

**Comment by michaelficarra** [comment]
Yeah, the two dots are not needed in CoffeeScript. That is a JavaScript only quirk. Should be easy to fix.
